notebooks/catvton/catvton.ipynb

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    convert_pipeline_models(pipeline, VAE_ENCODER_PATH, VAE_DECODER_PATH, UNET_PATH)
</pre><p>could you please provide opportunity to see conversion code? in description above you speak about convert_model, but there is no usage it in notebook. Please explain that for simplifying conversion process you use helper function that is based on convert_model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2453/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>